<launch>
  
  <node name="obj_to_pointcloud2" pkg="real_sim_registration" type="obj_to_pointcloud2" output="screen">
    <param name="map_frame_id" value="map" />
    <param name="obj_file_path" value="$(find real_sim_registration)/meshes/ee6f/ee6f.obj" />
    <param name="leaf_size" value="0.1" />
    <param name="publish_period" value="1.0" />
  </node>
</launch>
